locale.c

```diff
@@ -274,6 +274,16 @@
  *          correct results.  This is known to be a problem in earlier AIX
  *          versions
  *
+ *      -Accflags=-DHAS_BROKEN_LANGINFO_CODESET
+ *          This would be set in a hints file to tell perl that doing a libc
+ *              nl_langinfo(CODESET)
+ *          can give empty results, so perl instead gets compiled to use a
+ *          different method (known to be reliable) to determine if the current
+ *          locale is a UTF-8 one or not.  Perl continues to presume that a
```

Comment:
I'm confused by the sentence, `This would be set ... UTF-8 one or not.`  I concede that it's somewhat like the entry for `-Accflags=-DHAS_BROKEN_SETLOCALE_QUERY_LC_ALL`, but believe it would read more clearly if there were a period after `results` followed by a new sentence that does *not* begin with `so`.

Reply:
https://github.com/Perl/perl5/compare/c4c39eed64af46a140c255fb329fd8ca173e72ad..b50eff4a7723c2e2c0d6d43047de1f68f9e4cafc is an improvement.  Thanks.